xpti/include/xpti/xpti_trace_framework.hpp

```diff
@@ -353,6 +353,8 @@ class SpinLock {
 struct finally {
   std::function<void()> MFunc;
 
+  finally(const finally &) = default;
+  finally &operator=(const finally &) = default;
```

Comment:
I think there might be an argument to make this uncopiable instead to avoid accidental duplicate calls, given how this struct is used in the code right now.

Reply:
Ok, will do